tests/Unit/API/Management/JobsTest.php

```diff
@@ -23,7 +23,7 @@
 });
 
 test('createImportUsers() issues an appropriate request', function(): void {
-    $importPath = AUTH0_PHP_TEST_JSON_DIR . 'test-import-users-file.json';
+    $importPath = join(DIRECTORY_SEPARATOR, [AUTH0_TESTS_DIR, 'json', 'test-import-users-file.json']);
```

Comment:
`AUTH0_PHP_TEST_JSON_DIR` was an unnecessary global test declaration only used once; just getting rid of that here.